install_config/install/disconnected_install.adoc

```diff
@@ -379,9 +381,14 @@ name=rhel-7-server-extras-rpms
 baseurl=http://<server_IP>/repos/rhel-7-server-extras-rpms
 enabled=1
 gpgcheck=0
-[rhel-7-server-ose-3.4-rpms]
-name=rhel-7-server-ose-3.4-rpms
-baseurl=http://<server_IP>/repos/rhel-7-server-ose-3.4-rpms
+[rhel-7-fast-datapath-rpms]
+name=rhel-7-fast-datapath-rpms
+baseurl=http://<server_IP>/repos/rhel-7-fast-datapath-rpms
+enabled=1
+gpgcheck=0
+[rhel-7-server-ose-3.5-rpms]
+name=rhel-7-server-ose-3.5-rpms
+baseurl=http://<server_IP>/repos/rhel-7-server-ose-3.5-rpms
```

Comment:
@sdodson I added the fast-datapath channel to this Disconnected Installation topic here and a little above. It think it was all that was left related to https://github.com/openshift/openshift-docs/pull/3686.